dataflow/src/org/checkerframework/dataflow/cfg/DOTCFGVisualizer.java

```diff
@@ -397,12 +400,14 @@ public void visualizeBlockTransferInput(Block bb, Analysis<A, S, T> analysis) {
 
     @Override
     public void visualizeBlockNode(Node t, /*@Nullable*/ Analysis<A, S, T> analysis) {
-        A value = analysis.getValue(t);
-        String valueInfo = "";
-        if (value != null) {
-            valueInfo = "    > " + prepareString(value.toString());
+        this.sbBlock.append(prepareString(t.toString()) + "   [ " + prepareNodeType(t) + " ]");
+        if (analysis != null) {
+            A value = null;
+            value = analysis.getValue(t);
```

Comment:
Thanks. I had the same comment https://github.com/typetools/checker-framework/commit/2b100092d8cc2b339f7a93c6cedbd11267cae80e#commitcomment-18151499 but unfortunately that was buried within a commit.